### PR TITLE
Refactor python scripts

### PR DESCRIPTION
**Pull request by jhuntbach-bc**
No changes to the token files in this one, just refactoring the python scripts to make the next PR cleaner

### CODE REVIEW

File: scripts/grab-details-cryptocompare.py [hunk 1]
```diff
@@ -1,79 +1,30 @@
...
-        yield key, read_json(target)
+DATA_URL = 'https://min-api.cryptocompare.com/data/all/coinlist'
 
 
 def compress_duplicates(duplicates):
     return [(symbol, [x.name for x in group]) for symbol, group in duplicates]
 
 
-def _query_cryptocompare(url: str, errorCheck: bool = True, api_key: str = None) -> Optional[Dict]:
+def _query_cryptocompare(url: str, error_check: bool = True, api_key: str = None) -> Optional[Dict]:
```
Comment (jjambazov-bc) [inline scripts/grab-details-cryptocompare.py:22]: Please check feedback from https://github.com/blockchain/coin-definitions/pull/123#discussion_r1185897251

Comment (jhuntbach-bc) [reply scripts/grab-details-cryptocompare.py:22]: Addresses separately in https://github.com/blockchain/coin-definitions/pull/128

**Review by jjambazov-bc — APPROVED** [review]

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [ARG.CHZ, ERC20]: displayed as: ARG.CHZ
-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.007 not in the $0.01-$10 USD range
-  ⚠️  Coin(symbol='BSV', name='Bitcoin Satoshi Vision', key='bitcoinsv', decimals=8, logo=None, website=None): No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.004 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [POR.CHZ, ERC20]: displayed as: POR.CHZ
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC
-  ⚠️  [SOL, COIN]: minWithdrawal 0.1 -> $0.000 not in the $0.01-$10 USD range

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [ARG.CHZ, ERC20]: displayed as: ARG.CHZ
-  ⚠️  [BCH, COIN]: minWithdrawal 5460 -> $0.007 not in the $0.01-$10 USD range
-  ⚠️  Coin(symbol='BSV', name='Bitcoin Satoshi Vision', key='bitcoinsv', decimals=8, logo=None, website=None): No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.004 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [POR.CHZ, ERC20]: displayed as: POR.CHZ
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC
-  ⚠️  [SOL, COIN]: minWithdrawal 0.1 -> $0.000 not in the $0.01-$10 USD range

**Pull request merged**